streams/src/main/java/org/apache/kafka/streams/kstream/Consumed.java

```diff
@@ -70,6 +74,18 @@ private Consumed(final Serde<K> keySerde,
         this.processorName = processorName;
```

Comment:
This constructor should just set both variable:
```
@Deprecated
private Consumed(final Serde<K> keySerde,
                 final Serde<V> valueSerde,
                 final TimestampExtractor timestampExtractor,
                 final Topology.AutoOffsetReset legacyResetPolicy,
                 finalAutoOffsetReset resetPolicy,
                 final String processorName) {
    this.keySerde = keySerde;
    this.valueSerde = valueSerde;
    this.timestampExtractor = timestampExtractor;
    this.legacyResetPolicy = legacyResetPolicy;
    this.resetPolicy = resetPolicy;
    this.processorName = processorName;
}
```
This will help to simplify our internal code later.

Reply:
Added - see below comments, may need review. Thank you